gnomad/utils/constraint.py

```diff
@@ -1310,30 +1314,32 @@ def add_gencode_annotations(
         chromosome=gencode_ht.interval.start.contig,
     )
 
-    # Obtain transcript annotations from GENCODE file.
-    gencode_transcript = (
-        gencode_ht.filter(gencode_ht.feature == "transcript")
-        .select("chromosome", "transcript_id", "level", "transcript_type")
+    # Obtain CDS annotations from GENCODE file and calculate CDS length and
+    # number of exons.
+    annotations_to_add = annotations + ("chromosome", "transcript_id", "length")
```

Comment:
```suggestion
    annotations_to_add = set(annotations + ["chromosome", "transcript_id", "length"])
```
minor update to make the concatenation work + drop potentially duplicated annotations